src/systemctl/systemctl.c

```diff
@@ -4525,6 +4536,19 @@ static int show_one(
         if (r < 0)
                 return log_error_errno(r, "Failed to get properties: %s", bus_error_message(&error, r));
 
+        r = bus_message_map_all_properties(reply, property_map, &property_info);
+        if (r < 0)
+                return log_error_errno(r, "Failed to map properties: %s", bus_error_message(&error, r));
+
+        if (!isempty(property_info.load_state) && !isempty(property_info.active_state)) {
+                if (streq(property_info.load_state, "not-found") && streq(property_info.active_state, "inactive"))
```

Comment:
This outer condition can only be true if the inner condition is true. So you can simplify this to
```C
if (streq_ptr(property_info.load_state, "not-found") &&
    streq_ptr(property_info.active_state, "inactive"))
```